src/reports/package/accessibilityInsightsReport.d.ts

```diff
@@ -47,16 +47,50 @@ declare namespace AccessibilityInsightsReport {
     };
 
     export type SummaryReportParameters = {
-        serviceName: string;
-        axeVersion: string;
-        userAgent: string;
-        crawlDetails: CrawlSummaryDetails;
-        results: SummaryScanResults;
+        serviceName: string,
+        axeVersion: string,
+        userAgent: string,
+        crawlDetails: CrawlSummaryDetails,
+        results: SummaryScanResults,
     };
 
+    export type AxeRuleData = {
```

Comment:
This structure will not work for the give report mock. The minimum expected data from axe result per element:
```
{
    url,
    id[], // rules
    data[], // wcag
    html, // snippet
    target, // selector
    failureSummary, // how to fix
}
```
Please refer to [PM example](https://microsoft-my.sharepoint.com/:x:/r/personal/ferbonn_microsoft_com/_layouts/15/guestaccess.aspx?e=qbhTbR&share=Ed_NCGaoei9Eg2nFgpHeGqkBbGvcr3jNP9BzE1Uzja4rCA) and confirm if element may have multiple rules on a report as per PM example

<img width="779" alt="Screen Shot 2020-10-15 at 1 39 35 PM" src="https://user-images.githubusercontent.com/40576301/96185136-4fdfbb80-0eee-11eb-91c7-94dad94084f7.png">